service-storage/index.ts

```diff
@@ -63,6 +63,18 @@ app.post('/contracts/:contractAddress', async (req: Request, res: Response) => {
   }
 });
 
+app.delete('/contracts/:contractAddress', async (req: Request, res: Response) => {
+  const { contractAddress } = req.params;
+  try {
+    await delValue(contractAddress);
+    await deleteFile(`${contractAddress}.json`);
+    res.sendStatus(204);
+  } catch (err) {
+    console.error(err);
+    res.status(500).send(err);
+  }
+});
```

Comment:
Should we support not found case? it should return 404 instead of 500

Reply:
ok kub I will fix this krub